bears/natural_language/LanguageToolBear.py

```diff
@@ -25,12 +25,7 @@ def check_prerequisites(cls):
         if shutil.which('java') is None:
             return 'java is not installed.'
         else:
-            try:
-                from language_check import LanguageTool, correct
```

Comment:
@jayvdb , how about this 

```
try:
   from language_check import LanguageTool, correct
   LanguageTool;
   correct;
except ImportError:  # pragma: no cover		
   return 'Please install the `language-check` pip package.'
```
This would make pyflakes believe it is used as per [this](http://stackoverflow.com/a/10505551/5381204).